src/api/models.py

```diff
@@ -2,18 +2,256 @@
 
 db = SQLAlchemy()
 
-class User(db.Model):
+clientfavcenter = db.Table('clientfavcenter',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('center_id', db.Integer, db.ForeignKey('center.id'), primary_key=True)
+)
+
+
+clientfavspot = db.Table('clientfavspot',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('hotspot_id', db.Integer, db.ForeignKey('hotspot.id'), primary_key=True)
+)
+
+
+reviewspot = db.Table('reviewspot',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('hotspot_id', db.Integer, db.ForeignKey('hotspot.id'), primary_key=True)
+)
+
+
+reviewcenter = db.Table('reviewcenter',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('center_id', db.Integer, db.ForeignKey('center.id'), primary_key=True)
+)
+
+
+clientsport = db.Table('clientsport',
+    db.Column('client_id', db.Integer, db.ForeignKey('client.id'), primary_key=True),
+    db.Column('sport_id', db.Integer, db.ForeignKey('sport.id'), primary_key=True)
+)
+
+
+centersport = db.Table('centersport',
```

Comment:
los nombres los podéis poner siguiendo el standar de python client_sport, review_center...